src/features/jumpNav.tsx

```diff
@@ -254,8 +256,12 @@ const collapseReferenceChildren = () =>
         })
       );
       const li = Array.from(
-        document.querySelector(".bp3-popover-content > div> ul").children
-      ).find((e: HTMLLinkElement) => e.innerText === "Collapse all");
+        document.querySelector(
+          '.bp3-transition-container:not([style*="display: none;"]) .bp3-popover-content > div > ul'
```

Comment:
`.bp3-transition-container:not([style*="display: none;"])` this is the important addition to this PR

Another option could be `.bp3-portal:not[#unlink-custom-context-menu]`

Reply:
definitely prefer not putting unlink finder specific logic/selectors in workbench.

sucks that there is nothing else up the DOM tree to indicate that this dropdown is by Roam. This is the best we could do for now